pkg/operator/starter.go

```diff
@@ -46,7 +51,16 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	// Create GenericOperatorclient. This is used by the library-go controllers created down below
 	gvr := opv1.SchemeGroupVersion.WithResource("clustercsidrivers")
-	operatorClient, dynamicInformers, err := goc.NewClusterScopedOperatorClientWithConfigName(controllerConfig.KubeConfig, gvr, providerName)
+	gvk := opv1.SchemeGroupVersion.WithKind("ClusterCSIDriver")
+	operatorClient, dynamicInformers, err := goc.NewClusterScopedOperatorClientWithConfigName(
+		clock.RealClock{},
+		controllerConfig.KubeConfig,
+		gvr,
+		gvk,
+		providerName,
+		extractOperatorSpec,
+		extractOperatorStatus,
+	)
```

Comment:
The changes LGTM, but I suggest you split this commit into two: one for bumping the dependencies and another one to implement the changes required by the new dependencies (i.e., the changes in this file).